doc/source/release-notes/Percona-Server-5.6.34-79.1.rst

```diff
@@ -0,0 +1,54 @@
+.. rn:: 5.6.34-79.1
+
+============================
+|Percona Server| 5.6.34-79.1
+============================
+
+Percona is glad to announce the release of |Percona Server| 5.6.34-79.1 on
+November 21st, 2016 (Downloads are available `here
+<http://www.percona.com/downloads/Percona-Server-5.6/Percona-Server-5.6.34-79.1/>`_
+and from the :doc:`Percona Software Repositories </installation>`).
+
+Based on `MySQL 5.6.34
+<http://dev.mysql.com/doc/relnotes/mysql/5.6/en/news-5-6-34.html>`_, including
+all the bug fixes in it, |Percona Server| 5.6.34-79.1 is the current GA release
+in the |Percona Server| 5.6 series. All of |Percona|'s software is open-source
+and free, all the details of the release can be found in the `5.6.34-79.1
+milestone at Launchpad
+<https://launchpad.net/percona-server/+milestone/5.6.34-79.1>`_.
+
+New Features
+============
+
+ MyRocks code has been imported into |Percona Server| as experimental. It is
```

Comment:
Since it will not even compile properly in the current state, I do not think we should mention it at all. Lets wait until we are at a point where it will compile and pass some/most mtr tests where we can formally reveal it and declare it as experimental.